pkg/querier/querier.go

```diff
@@ -51,9 +51,11 @@ type Config struct {
 
 	ShuffleShardingIngestersEnabled bool `yaml:"shuffle_sharding_ingesters_enabled" category:"advanced"`
 
-	PreferStreamingChunks                      bool   `yaml:"prefer_streaming_chunks" category:"experimental"`
-	StreamingChunksPerIngesterSeriesBufferSize uint64 `yaml:"streaming_chunks_per_ingester_series_buffer_size" category:"experimental"`
-	MinimizeIngesterRequests                   bool   `yaml:"minimize_ingester_requests" category:"experimental"`
+	PreferStreamingChunks                                  bool   `yaml:"prefer_streaming_chunks" category:"experimental"`
+	PreferStreamingChunksStoregateway                      bool   `yaml:"prefer_streaming_chunks_storegateway" category:"experimental"`
+	StreamingChunksPerIngesterSeriesBufferSize             uint64 `yaml:"streaming_chunks_per_ingester_series_buffer_size" category:"experimental"`
+	StreamingChunksPerIngesterSeriesBufferSizeStoregateway uint64 `yaml:"streaming_chunks_per_ingester_series_buffer_size_storegateway" category:"experimental"`
```

Comment:
Given the existing flags are marked as experimental, should we take the pain now of renaming `PreferStreamingChunks` (and the corresponding YAML option and CLI flag) to make it clear it's for ingester streaming?

Also, I believe our preferred terminology is to treat "store-gateway" as two words:

```suggestion
	PreferStreamingChunksFromIngesters                                  bool   `yaml:"prefer_streaming_chunks_from_ingesters" category:"experimental"`
	PreferStreamingChunksFromStoreGateways                      bool   `yaml:"prefer_streaming_chunks_from_store_gateways" category:"experimental"`
	StreamingChunksPerIngesterSeriesBufferSize             uint64 `yaml:"streaming_chunks_per_ingester_series_buffer_size" category:"experimental"`
	StreamingChunksPerStoreGatewaySeriesBufferSize uint64 `yaml:"streaming_chunks_per_store_gateway_series_buffer_size" category:"experimental"`
```

Reply:
> should we take the pain now of renaming PreferStreamingChunks

Does not sounds like a bad idea